### DISCUSSION TITLE
how to connect with db

### DISCUSSION THREAD

**Discussion by ghost** [Interactions-as-a-Service/d1-orm#76] [Q&A] [answered]
![image](https://github.com/Interactions-as-a-Service/d1-orm/assets/155236620/ae1ddaea-2bf4-4a18-9547-2fed338a434f)
I just put env.DB (the database id) that's it ? no secret key or something to connect to db?
I want to use it with my NestJS / Exprses js node backend.

**Comment by Skye-31** [comment, marked as answer]
This library is for use in Workers, where `env.DB` has already been authenticated for you. We do not support usage in Node
